src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -1,12 +1,18 @@
 package org.influxdb.impl;
 
 
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.MappingJsonFactory;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
```

Comment:
Same here, i really would like to prevent to switch from moshi to jackson with this PR. I am not against changing the json converter library, i just want to do it by "accident" when introducing messagepack format. We should do this move, once decided intentionally and without any other modifications.

Reply:
@majst01 I believe Jackson is being used here because it's a direct dependency used by the msgpack converter library, as you can see [here](https://github.com/komamitsu/retrofit-converter-msgpack/blob/master/build.gradle#L36).

Reply:
again, the convertor depends on Jackson